Exec/hydro_tests/riemann_2d/problem_initialize_state_data.H

```diff
@@ -0,0 +1,95 @@
+#ifndef problem_initialize_state_data_H
+#define problem_initialize_state_data_H
+
+//The _prob_parameters file is parsed by this library
+#include <prob_parameters.H>
+
+//The gamma_law is introduced by the following library
+#include <eos.H>
+
+AMREX_GPU_HOST_DEVICE AMREX_INLINE
+//Declare boxes index i, j, k, the state Array4<Real>, and 
+//the geometry information, which emcompases the  
+void problem_initialize_state_data(int i, int j, int k, 
+		                   Array4<Real> const& state,
+				   const GeometryData& geomdata)
+{
+	//The integer "coord_type" represents the coordinate system
+	int coord_type = geomdata.Coord(); 
+	
+	//The pointer *dx represents the real cell size value 
+	const Real* dx = geomdata.CellSize();
+
+	//The pointer *problo represents the bottom of each dimension
+	const Real* problo = geomdata.ProbLo();
+
+#if AMREX_SPACEDIM != 2
+	amrex::Abort("The Riemman Problem considered is 2D.");
+	
+#endif
+
+
+#if AMREX_SPACEDIM == 2
+	Real xx = problo[0] + dx[0] * (static_cast<Real>(i) + 0.5_rt);
+	Real yy = problo[1] + dx[1] * (static_cast<Real>(j) + 0.5_rt);
+#endif
+
+	//Now we want to fracture the domain in 4 regions as described in
+	//Liska & Wendroff (2003).
+	if (xx < problem::center[0] && yy < problem::center[1])
```

Comment:
can you add a comment just to help make it clear which quadrant is which? like just doing
```
// 2|1
// -+-
// 3|4
```